chat_ui/src/main/java/io/skygear/plugins/chat/ui/ConversationFragment.kt

```diff
@@ -136,6 +137,29 @@ open class ConversationFragment() :
         arguments.getSerializable(ConnectionListenerKey)?.let { listener ->
             connectionListener = listener as ConnectionListener
         }
+
+        this.pubsubListener = object : PubsubListener {
+            override fun onClose() {
+                needRefresh = true
+                if (this@ConversationFragment.isResumed()) {
```

Comment:
I think the condition `this@ConversationFragment.isResumed()` should be removed, because:

- `connectionListener` sounds purely related to connection state, so i think the one who use this variable does not expect it is also related to the fragment lifecycle
- `connectionListener` is a protected variable, so subclassed fragment can also access the `isResumed`

Reply:
Briefly talked with @carmenlau, `connectionListener` is supposed to be assigned from the previous activity.

And the original example would show toasts in the listener events, which I think it's a bit weird and misleading. Since we are not trying to change the behaviour in the example and we can't think of a meaningful use case for the listener yet, I agree to keep the old behaviour and keep  the `this@ConversationFragment.isResumed()`.